apps/orders/models.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+
+from apps.products.models import Products
+from apps.users.models import User, UserAddress
+
+
+# Create your models here.
+
+class BaseModelMixin(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+
+
+class Cart(BaseModelMixin):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='cart')
+    is_archived = models.BooleanField(default=False)
+
+
+class CartItem(BaseModelMixin):
+    product = models.ForeignKey(Products, on_delete=models.CASCADE, related_name='carts')
+    cart = models.ForeignKey(Cart, on_delete=models.CASCADE, related_name='items')
+    price = models.DecimalField(max_digits=6, decimal_places=2)
+    discount = models.IntegerField(validators=[MinValueValidator(1), MaxValueValidator(99)])
```

Comment:
de ce salvezi aici discount daca salvezi discount in product?

Reply:
Productul dupa un timp poate sa nu mai fie la reducere, deaceasta cind se v-a face checkout se va pastra reducerea din timpul curent.